daemon/daemon.go

```diff
@@ -850,34 +873,40 @@ func (daemon *Daemon) Graph() *graph.Graph {
 	return daemon.graph
 }
 
+// Repositories returns all repositories.
 func (daemon *Daemon) Repositories() *graph.TagStore {
 	return daemon.repositories
 }
 
-func (daemon *Daemon) Config() *Config {
-	return daemon.config
+func (daemon *Daemon) config() *Config {
+	return daemon.configStore
 }
 
-func (daemon *Daemon) SystemConfig() *sysinfo.SysInfo {
+func (daemon *Daemon) systemConfig() *sysinfo.SysInfo {
 	return daemon.sysInfo
 }
 
-func (daemon *Daemon) SystemInitPath() string {
+func (daemon *Daemon) systemInitPath() string {
 	return daemon.sysInitPath
 }
 
+// GraphDriver returns the currently used driver for processing
+// container layers.
 func (daemon *Daemon) GraphDriver() graphdriver.Driver {
 	return daemon.driver
 }
 
+// ExecutionDriver returns the currently used driver for creating and
+// starting execs in a container.
 func (daemon *Daemon) ExecutionDriver() execdriver.Driver {
 	return daemon.execDriver
 }
 
-func (daemon *Daemon) ContainerGraph() *graphdb.Database {
-	return daemon.containerGraph
+func (daemon *Daemon) containerGraph() *graphdb.Database {
+	return daemon.containerGraphDB
 }
 
+// ImageGetCached return an image created before the given image with the same configuration.
```

Comment:
s/return/returns/

What does it mean for an image to be created before another image? I feel like this could be clarified.